src/Akeneo/Category/back/Application/Command/DeactivateTemplateCommand.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Category\Application\Command;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @copyright 2023 Akeneo SAS (https://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class DeactivateTemplateCommand
+{
+    public function __construct(
```

Comment:
```suggestion
    private function __construct(
```
Either make the constructor private or remove the static constructor